NitroxClient/Communication/PacketSender.cs

```diff
@@ -56,6 +56,12 @@ public void PickupItem(GameObject gameObject, String techType)
             String guid = GuidHelper.GetGuid(gameObject);
             Vector3 itemPosition = gameObject.transform.position;
 
+            PickupItem(itemPosition, guid, techType);
```

Comment:
This looks like it will send the message twice. (Once in PickupItem and then on the next line)